.devcontainer/dockerBuild.sh

```diff
@@ -0,0 +1,7 @@
+# add poetry to PATH
+export PATH="/root/.poetry/bin:$PATH"
```

Comment:
I don't think this is the best place for this.  If you open a new terminal it won't run this so poetry won't be available in the path.